### DISCUSSION TITLE
AAB targets API level 32, but Play Store minimum is currently level 33

### DISCUSSION THREAD

**Discussion by codingthat** [html2app/discussions#14] [Q&A] [answered]
Uploaded a signed release aab to "Internal Testing" in the GP console, and got this:

> 
> Error
> 
> Your app currently targets API level 32 and must target at least API level 33 to ensure it is built on the latest APIs optimized for security and performance. [Learn More](https://developer.android.com/google/play/requirements/target-sdk)

Any possibility to target 33+ ? :pray:

**Comment by yandeu** [comment, marked as answer]
Done, please try to build it again.

**Reply by yandeu** [reply]
Did it work?

**Reply by codingthat** [reply]
Worked great, thank you very much for the update!  :pray:

**Comment by codingthat** [comment]
Ah, similarly, they've now raised the minimum to level 34.  :pray:

**Reply by codingthat** [reply]
(@yandeu Maybe this would make sense as a `config.json` option?  :) )

**Reply by yandeu** [reply]
I have updated the level to 34.

> Maybe this would make sense as a `config.json` option?

If I add this, it would be very hard for me to help debugging if the pipeline fails. I prefer setting it manually.
